tests/integration/pom.xml

```diff
@@ -99,27 +99,31 @@
             <groupId>org.mule.modules</groupId>
             <artifactId>mule-module-validation</artifactId>
             <version>${project.version}</version>
+            <classifier>mule-plugin</classifier>
```

Comment:
it should but as the mule-module-file-extension-common renaming will be part of MULE-11383 (all artifacts will become JAR files) it's left as it is right now